lib/src/gestures/gestures.dart

```diff
@@ -12,7 +12,7 @@ import 'package:positioned_tap_detector_2/positioned_tap_detector_2.dart';
 
 abstract class MapGestureMixin extends State<FlutterMap>
     with TickerProviderStateMixin {
-  static const double _kMinFlingVelocity = 800.0;
+  static const double _kMinFlingVelocity = 800;
```

Comment:
Same here?


```suggestion
  static const int _kMinFlingVelocity = 800;
```

Reply:
In this case it's fine for this to be an int since it's only used in one place as a condition check.